release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1312,6 +1312,10 @@ With this release, replacement control plane nodes are assigned to the correct i
 [id="ocp-4-13-scalability-and-performance-bug-fixes"]
 ==== Scalability and performance
 
+* Previously, when a performance profile was generated, the CRI-O runtime files created automatically were configured to use `runc` as the CRI-O runtime.
++
+Now that setting `crun` as the container runtime is generally available when a performance profile is generated the runtime CRI-O files created match the `defaultRuntime` configured in the `ContainerRuntimeConfig` CR. This can be either `crun` or `runc`. The default is `runc`. (link:https://issues.redhat.com/browse/OCPBUGS-11813[*OCPBUGS-11813*])
```

Comment:
```suggestion
Now that setting `crun` as the container runtime is generally available when a performance profile is generated, the runtime CRI-O files created match the `defaultRuntime` configured in the `ContainerRuntimeConfig` CR. This can be either `crun` or `runc`. The default is `runc`. (link:https://issues.redhat.com/browse/OCPBUGS-11813[*OCPBUGS-11813*])
```

Reply:
a comma is needed after the phrase "Now that setting `crun` as the container..."